lib/hyacinth/digital_object/asset_type.rb

```diff
@@ -0,0 +1,25 @@
+module Hyacinth
+  module DigitalObject
+    module AssetType
```

Comment:
We may have to update the best_type gem to accommodate some of these, but if so, that's a good improvement.